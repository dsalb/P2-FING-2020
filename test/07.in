hacerVecinos 1 10 100
hacerVecinos 2 9 100
hacerVecinos 3 8 100
hacerVecinos 4 7 100
hacerVecinos 5 6 100
sonVecinos 1 10
sonVecinos 3 9
sonVecinos 4 8
sonVecinos 5 6
accesibles 1
accesibles 2
accesibles 3
accesibles 4
accesibles 5
accesibles 6
accesibles 7
accesibles 8
accesibles 9
accesibles 10
Fin

