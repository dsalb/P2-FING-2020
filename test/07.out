1>Nuevos vecinos: 1 10 100.00.
2>Nuevos vecinos: 2 9 100.00.
3>Nuevos vecinos: 3 8 100.00.
4>Nuevos vecinos: 4 7 100.00.
5>Nuevos vecinos: 5 6 100.00.
6>Son vecinos.
7>NO son vecinos.
8>NO son vecinos.
9>Son vecinos.
10>1 0 0 0 0 0 0 0 0 1 
11>0 1 0 0 0 0 0 0 1 0 
12>0 0 1 0 0 0 0 1 0 0 
13>0 0 0 1 0 0 1 0 0 0 
14>0 0 0 0 1 1 0 0 0 0 
15>0 0 0 0 1 1 0 0 0 0 
16>0 0 0 1 0 0 1 0 0 0 
17>0 0 1 0 0 0 0 1 0 0 
18>0 1 0 0 0 0 0 0 1 0 
19>1 0 0 0 0 0 0 0 0 1 
20>Fin.
